db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+ancient_aliens = Meme.create(name: 'Ancient Aliens', image_url: 'http://i3.kym-cdn.com/photos/images/newsfeed/000/158/326/9148130.jpg', description: 'I don\'t know, therefore aliens.')
```

Comment:
Follow these comments. You dont need to assign a variable to your Meme creation, remove the variable. 
And destructure this line if possible as well, its a tad long)